connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -326,12 +321,18 @@ public static void logout() {
     }
 
     public static synchronized void sdkInitialize(Context applicationContext) {
+        sdkInitialize(applicationContext, false);
```

Comment:
Change `false` -> `true`.

Reply:
Ok, but that breaks the test case `getConnectApiUrlReturnsProductionByDefault()`. Should I change the test as well?

Reply:
You can delete that whole test. The method used to be in the `ConnectSdk` class, but since it got moved to `ConnectUrlHelper` it doesn't really belong there anymore. And the method is pretty trivial, so should be fine to delete it.

Reply:
Done.